packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2232,6 +2232,38 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
       return;
     assert(() {
       if (_needsCompositingBitsUpdate) {
+        final RenderObject parent = this.parent as RenderObject;
+        if (parent is RenderObject) {
+          bool isChildOfParent = false;
```

Comment:
Maybe call this `isVisitedByParent`?  Otherwise it may be confusing to read since the child _is_ a child of its parent.